code/modules/tgui/tgui_window.dm

```diff
@@ -311,6 +311,18 @@
 			: "[id].browser:update")
 	message_queue = null
 
+/**
+ * public
+ *
+ * Replaces the inline HTML content.
+ *
+ * required inline_html string HTML to inject
+ */
+/datum/tgui_window/proc/replace_html(inline_html)
+	if(!inline_html)
+		return
+	send_message("replace_html", inline_html)
```

Comment:
I'd say you should do something like this, to allow completely replacing html with nothing.

```suggestion
/datum/tgui_window/proc/replace_html(inline_html = "")
	send_message("replace_html", inline_html)
```